packages/user-service/src/typedef.graphql

```diff
@@ -127,8 +127,8 @@ type Mutation  {
 }
 
 type Query  {
-  # Fetches specific User by id
-  getUser(uid: String!): UserType
+  # Fetches specific User by uid, rhatUUID, apiRole, name
+  getUsersBy( _id:String, uid: String, rhatUUID: String, apiRole: String, name: String): [UserType]
```

Comment:
Can you change the _id to id? We shouldn't expose the _id as an external parameter/input.

```suggestion
  getUsersBy( id: String, uid: String, rhatUUID: String, apiRole: String, name: String): [UserType]
```